### PR TITLE
DA3C losses & entropies

### PR DESCRIPTION
**Pull request by 4SkyNet**

### CODE REVIEW

File: docs/Algorithms.md [hunk 1]
```diff
@@ -228,8 +228,11 @@ You must specify the parameters for the algorithm in the corresponding `app.yaml
         epsilon: 0.1
 
     entropy_beta: 0.01              # entropy regularization constant
+    entropy_type: Origin            # choice within Normal `Gauss` and `Origin` A3C entropy
     rewards_gamma: 0.99             # rewards discount factor
     gradients_norm_clipping: 40.    # value for gradients norm clipping
+    policy_clip: false              # it allows to clip policy loss within range [-value, +value]
```
Comment (StanislavVolodarskiy) [inline docs/Algorithms.md]: Where is value defined?

File: relaax/algorithms/da3c/da3c_continuous_model.py [hunk 2]
```diff
@@ -168,11 +168,11 @@ def build_graph(self):
             feeds.update(dict(action=sg_icm_network.ph_taken))
             self.op_compute_icm_gradients = self.Op(sg_icm_gradients.calculate, **feeds)
 
-        batch_size = tf.to_float(tf.shape(sg_network.ph_state.node)[0])
+        # batch_size = tf.to_float(tf.shape(sg_network.ph_state.node)[0])
```
Comment (StanislavVolodarskiy) [inline relaax/algorithms/da3c/da3c_continuous_model.py]: Please remove.

File: relaax/common/algorithms/lib/loss.py [hunk 3]
```diff
@@ -66,12 +66,22 @@ def build_graph(self, actor, critic, cfg):
         normal_dist = tf.contrib.distributions.Normal(mu, sigma2)
         log_prob = normal_dist.log_prob(self.ph_action.node)
 
-        self.entropy = tf.reduce_mean(-0.5 * (tf.log(2 * np.pi * sigma2) + 1.0))
+        if cfg.entropy_type == 'Gauss':
+            self.entropy = tf.reduce_mean(normal_dist.entropy())
+        elif cfg.entropy_type == 'Origin':
+            self.entropy = tf.reduce_mean(-0.5 * (tf.log(2 * np.pi * sigma2) + 1.0))
+        else:
+            assert True, 'You should provide entropy type from 2 variants: Gauss or Origin'
```
Comment (StanislavVolodarskiy) [inline relaax/common/algorithms/lib/loss.py]: `assert True` does nothing.
